tests/unit/email-dropdown.spec.js

```diff
@@ -112,6 +112,21 @@ describe("EmailDropdown.vue", () => {
       expect(wrapper.emitted().input[1]).to.have.length(1);
       expect(wrapper.emitted().input[1][0]).to.be.equal("hello@gmail.");
     });
+
+    it("returns 'true' if input is empty after clicking clean (X) button", async () => {
```

Comment:
Also, this test was created within the `events` describe, this should be placed on its own `describe` in the root of the test file.